examples/spdx2_document_from_scratch.py

```diff
@@ -0,0 +1,145 @@
+    ],
+)
+
+# Now that we have a package defined, we can add it to the document's package property.
+document.packages = [package]
+
+# A DESCRIBES relationship asserts that the document indeed describes the package.
+describes_relationship = Relationship("SPDXRef-Document", RelationshipType.DESCRIBES, "SPDXRef-Package")
+document.relationships = [describes_relationship]
+
+# Let's add two files. Have a look at the file class for all possible properties a file can have.
+file1 = File(
+    name="./package/file1.py",
+    spdx_id="SPDXRef-File1",
+    file_types=[FileType.SOURCE],
+    checksums=[
+        Checksum(ChecksumAlgorithm.SHA1, "d6a770ba38583ed4bb4525bd96e50461655d2758"),
+        Checksum(ChecksumAlgorithm.MD5, "624c1abb3664f4b35547e7c73864ad24"),
+    ],
+    license_concluded=get_spdx_licensing().parse("MIT"),
+    license_info_in_file=[get_spdx_licensing().parse("MIT")],
+    copyright_text="Copyright 2022 Jane Doe",
+)
+file2 = File(
+    name="./package/file2.py",
+    spdx_id="SPDXRef-File2",
+    checksums=[
+        Checksum(ChecksumAlgorithm.SHA1, "d6a770ba38583ed4bb4525bd96e50461655d2759"),
+    ],
+    license_concluded=get_spdx_licensing().parse("GPL-2.0-only"),
+)
+
+# Assuming the package contains those two files, we create two CONTAINS relationships.
+contains_relationship1 = Relationship("SPDXRef-Package", RelationshipType.CONTAINS, "SPDXRef-File1")
+contains_relationship2 = Relationship("SPDXRef-Package", RelationshipType.CONTAINS, "SPDXRef-File2")
+
+# This library uses run-time type checks when assigning properties.
+# Because in-place alterations like .append() circumvent these checks, we don't use them here.
+document.relationships += [contains_relationship1, contains_relationship2]
+
+# We now have created a document with basic creation information, describing a package that contains two files.
+# You can also add Annotations, Snippets and ExtractedLicensingInfo to the document in an analogous manner to the above.
+# Have a look at their respective classes if you are unsure about their properties.
+
+
+# This library provides comprehensive validation against the SPDX specification.
+# Note that details of the validation depend on the SPDX version of the document.
+validation_messages: List[ValidationMessage] = validate_full_spdx_document(document)
+
+# If the document is valid, validation_messages will be empty.
+assert validation_messages == []
+
+# Else, you can have a look at each entry's message and context
+# which will help you pinpoint the location of the invalidity.
+for message in validation_messages:
+    logging.warning(message.validation_message)
+    logging.warning(message.context.spdx_id)
+
+# Finally, we can serialize the document to any of the five supported formats.
+# Using the write_file() method from the write_anything module,
+# the format will be determined by the file ending: .spdx (tag-value), .json, .xml, .yaml. or .rdf (or .rdf.xml)
+write_file(document, "my_spdx_document.json")
```

Comment:
According to the specification, JSON serializations should use `*.spdx.json` (see https://spdx.github.io/spdx-spec/v2.3/conformance/#44-standard-data-format-requirements), so I would probably stick with the recommendation here as well?